### ISSUE TITLE
Middleware decorator make case sensitive routing error

### ISSUE THREAD

**Issue by AlbertAZ1992** [eggjs/tegg#15]
## What happens?
Middleware decorator cannot make case sensitive routing. 

It seems that `getEggObject` will get wrong ObjectMap after Middleware decorator.
```
eggObjectMap Map(3) {
  'LOAD_UNIT:tegg-app-controller:PROTO:853:middlewareController' => Map(1) {  },
  'LOAD_UNIT:multiModuleService:PROTO:851:appService' => Map(1) {},
  'LOAD_UNIT:multi-module-repo:PROTO:850:appRepo' => Map(1) {}
}
proto.id: LOAD_UNIT:tegg-app-controller:PROTO:857:viewController
```

## Mini Showcase Repository(REQUIRED)
> Provide a mini GitHub repository which can reproduce the issue.
> Use `npm init egg --type=simple bug` then upload to your GitHub
 
add following into `plugin/controller/test/http/middleware.test.ts`
```
  it('method Middleware should not work', async () => {
    app.mockCsrf();
    await app.httpRequest()
      .get('/Middleware/Method')
      .expect(200)
      .expect('hello, view');
  });
```

## How To Reproduce

**Steps to reproduce the behavior:**
1. Use middleware with HttpController
```
@HTTPController({
  path: '/middleware',
})
@Middleware(countMw)
export class MiddlewareController {
  @Inject()
  appService: AppService;

  @HTTPMethod({
    method: HTTPMethodEnum.GET,
    path: '/global',
  })
  async global() {
    return {};
  }

  @HTTPMethod({
    method: HTTPMethodEnum.GET,
    path: '/method',
  })
  @Middleware(logMwFactory('use middleware'))
  async middleware() {
    return {};
  }
}
```
add a viewController to match other paths
```
@HTTPController()
export class ViewController {

  @HTTPMethod({
    method: HTTPMethodEnum.GET,
    path: '/*',
  })
  async get() {
    return 'hello, view';
  }
}
```
2. try to visit a Uppercase of router path : `/Middleware/Method`，response 500 server error

**Expected behavior**
1. router match with `/*` and response 'hello, view'

## Context
- **Node Version**:
- **Egg Version**:
- **Plugin Name**:
- **Plugin Version**:
- **Platform**:

**Comment by killagu** [comment]
@AlbertAZ1992 Thanks for the issue. This bug has beed solved with `+ @eggjs/tegg-controller-plugin@0.1.14`.

**Comment by AlbertAZ1992** [comment]
> 



> @AlbertAZ1992 Thanks for the issue. This bug has beed solved with `+ @eggjs/tegg-controller-plugin@0.1.14`.

@killagu I got some ERROR after added `@eggjs/tegg-controller-plugin@0.1.14`. I think we need more update: https://github.com/eggjs/tegg/pull/20
